package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "openpgp",
   "description": "OpenPGP.js is a Javascript implementation of the OpenPGP protocol. This is defined in RFC 4880.",
-  "version": "1.1.0",
+  "version": "1.0.1",
```

Comment:
No need to Update the version number. As this happens automatically when I do a release.